.github/workflows/sam-app-post-merge-actions.yaml

```diff
@@ -56,6 +56,11 @@ jobs:
       - name: Test query-activity-log
         run: npm ci && npm test
         working-directory: ./lambda/query-activity-log
+      
+      - name: Test write-activity-log
+        run: npm ci && npm test
+        working-directory: ./lambda/write-activity-log
+
```

Comment:
Could we please include  write-activity-log to the sonar workflows as well

Reply:
Oh that's a good catch, I'd be happy for this to go in here, or to make this a follow up task.
I know merging this unblocks you on delivery a bit right Dougal?

And there's nothing much we lose from adding this say... later this week over ensuring it's there at the start right @divyaparameswaran?

So long as we don't forget I'd be happy. I'm still getting relatively variable mileage out of using sonar 🤔